crates/iota-graphql-rpc/src/server/builder.rs

```diff
@@ -586,6 +580,35 @@ async fn graphql_handler(
     (extensions, result.into())
 }
 
+pub(crate) fn get_write_api<'ctx>(
+    ctx: &'ctx Context<'_>,
+) -> Result<&'ctx OptimisticWriteApi, Error> {
+    let write_api: &Option<OptimisticWriteApi> = ctx
+        .data()
+        .map_err(|_| Error::Internal("Unable to fetch IOTA SDK client".to_string()))?;
```

Comment:
Error messages in this function should be adjusted, this is no longer an SDK client.

Reply:
7bca8228c0